android/src/main/java/com/audiowaveform/AudioWaveformModule.kt

```diff
@@ -316,7 +316,7 @@ class AudioWaveformModule(context: ReactApplicationContext): ReactContextBaseJav
                     }
                 }
                 override fun onReject(error: String?, message: String?) {
-                    promise.reject(error, message)
+                    promise.reject(error ?: "Error", message ?: "An error is thrown while decoding the audio file")
```

Comment:
If we switch aguments of [this code](https://github.com/SimformSolutionsPvtLtd/react-native-audio-waveform/blob/de8db716aa9bde3220cee052639e4a4241ec058d/android/src/main/java/com/audiowaveform/WaveformExtractor.kt#L167-L168),
We can change the signature to:
```
override fun onReject(error: String, message: String?)
```